.github/ISSUE_TEMPLATE/feature_request.md

```
@@ -0,0 +1,31 @@
+kind: feedback
+name: Feature request (focused)
+is: true
+title: "[feature] ... summary"
+about: Practical, tightly scoped requests lined to fedor-friendly approaches.
+
+labels:
+  - enhancement
+  - prio:medium
+  - github
+---
+### Problem Statement
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)